tests/compiler/features/reference-types.optimized.wat

```diff
@@ -1,25 +1,25 @@
 (module
- (type $anyref_=>_anyref (func (param anyref) (result anyref)))
+ (type $externref_=>_externref (func (param externref) (result externref)))
  (type $none_=>_none (func))
  (type $i32_i32_i32_i32_=>_none (func (param i32 i32 i32 i32)))
- (type $anyref_=>_none (func (param anyref)))
- (type $anyref_anyref_=>_i32 (func (param anyref anyref) (result i32)))
- (type $anyref_anyref_=>_anyref (func (param anyref anyref) (result anyref)))
- (import "reference-types" "someObject" (global $features/reference-types/someObject anyref))
- (import "reference-types" "someKey" (global $features/reference-types/someKey anyref))
- (import "Reflect" "has" (func $~lib/bindings/Reflect/has (param anyref anyref) (result i32)))
+ (type $externref_=>_none (func (param externref)))
+ (type $externref_externref_=>_i32 (func (param externref externref) (result i32)))
+ (type $externref_externref_=>_externref (func (param externref externref) (result externref)))
+ (import "reference-types" "someObject" (global $features/reference-types/someObject externref))
+ (import "reference-types" "someKey" (global $features/reference-types/someKey externref))
+ (import "Reflect" "has" (func $~lib/bindings/Reflect/has (param externref externref) (result i32)))
  (import "env" "abort" (func $~lib/builtins/abort (param i32 i32 i32 i32)))
- (import "console" "log" (func $~lib/bindings/console/log (param anyref)))
- (import "Reflect" "get" (func $~lib/bindings/Reflect/get (param anyref anyref) (result anyref)))
- (import "reference-types" "external" (func $features/reference-types/external (param anyref) (result anyref)))
+ (import "console" "log" (func $~lib/bindings/console/log (param externref)))
+ (import "Reflect" "get" (func $~lib/bindings/Reflect/get (param externref externref) (result externref)))
+ (import "reference-types" "external" (func $features/reference-types/external (param externref) (result externref)))
  (memory $0 1)
  (data (i32.const 1024) "6\00\00\00\01\00\00\00\01\00\00\006\00\00\00f\00e\00a\00t\00u\00r\00e\00s\00/\00r\00e\00f\00e\00r\00e\00n\00c\00e\00-\00t\00y\00p\00e\00s\00.\00t\00s")
- (global $features/reference-types/nullGlobal (mut anyref) (ref.null))
+ (global $features/reference-types/nullGlobal (mut externref) (ref.null))
```

Comment:
Looking through the fixtures, this is one such occasion introducing a breaking change for text format. Here, `anyref` has already been renamed to `externref`, but it is used with `ref.null`, which is now invalid iiuc.

Reply:
I guess we could just comment or remove this code parts for now. WDYT?

Reply:
The issue here is that updating Binaryen makes this a breaking change, so my preference would be to tackle https://github.com/AssemblyScript/assemblyscript/pull/1381 first, even if in a preliminary way, before updating to a version of Binaryen with renamed `externref`. Hmm...

Reply:
> my preference would be to tackle #1381 first

Did you mean #1319 ?